newapi/ooniapi/utils.py

```diff
@@ -22,3 +23,23 @@ def nocachejson(*a, **kw):
     resp.cache_control.max_age = 0
     resp.cache_control.no_cache = True
     return resp
+
+
+def jer(error_id, tpl, *a, **kw) -> Response:
+    """Error handler with localization support
+    Return error_id, error_msg human friendly msg and errdata dict
+    Set Cache-Control: No-Cache
+    Set HTTP error code from the first optional argument or default to 400
+    """
+    # The function definition is parsed by ../scan_error_handlers.py
+    if error_id == "":
+        error_id = tpl.title().replace(" ", "")
```

Comment:
I don't think generating the ID from the template it's a good idea. It seems very non robust and semantically strange. So I can anticipate someone else who's not familiar with these subtle details changing the template and obtaining, as a result, also a different error ID. So, I suggest not to use this surprising pattern. Rather, it would be better, in my opinion, to explicitly mention the correct error ID each time.